benchmarks/python/test_huggingface_attn_fwd.py

```diff
@@ -71,9 +71,26 @@ def huggingface_attn_fwd_fusion(
     fd.add_output(T47)
 
 
+def huggingface_attn_fwd_fn(
```

Comment:
do we have references for this and all the other baseline implementations?

Reply:
I'll add links to the thunder code: They were removed earlier. 
Is that what you're referring to?